### PR TITLE
Add matrix for storage backend testing, SDA

### PR DESCRIPTION
**Pull request by jbygdell**
* Adds matrix for storage type to the SDA test Action
* Adds a docker compose file for testing posix storage
* Updates the required shell scripts

### CODE REVIEW

File: .github/integration/scripts/make_certs.sh [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 
-# install openssl if it's missing
-if [ ! "$(command -v openssl)" ]; then
-    if [ "$(id -u)" != 0 ]; then
-        echo "openssl is missing, unable to install it"
-        exit 1
-    fi
+# install requirements if it's missing
+for r in openssl openjdk8-jre-base; do
+    if [ ! "$(command -v openssl)" ]; then
```
Comment (pontus) [inline .github/integration/scripts/make_certs.sh:6]:
```suggestion
    if [ ! "$(command -v "$r")" ]; then
```
? Even though openjdk8-jre-base seems an unexpected name to me.

File: .github/integration/scripts/make_sda_credentials.sh [hunk 2]
```diff
@@ -1,28 +1,34 @@
 #!/bin/sh
 set -e
 
+URI=http://rabbitmq:15672
+if [ -n "$PGSSLCERT" ]; then
```
Comment (pontus) [inline .github/integration/scripts/make_sda_credentials.sh:5]: ?

Comment (jbygdell) [reply .github/integration/scripts/make_sda_credentials.sh:5]: Postgres requires a client certificate, so it's the simplest way of detecting if TLS is enabled or not.

Comment (pontus) [reply .github/integration/scripts/make_sda_credentials.sh:5]: Fine enough, but could we get a comment highlighting that postgres tls usage is taken as proxy for tls usage for others?

**Review by pontus — DISMISSED** [review]

**Review by aaperis — APPROVED** [review]
Looks good :-)

**Review by pontus — APPROVED** [review]

**Review by aaperis — APPROVED** [review]

**Pull request merged**